concepts/arrays/about.md

```diff
@@ -137,6 +142,76 @@ Array.isArray(object);
 You might be tempted to use `names instanceof Array`, and that can work, but not under all circumstances.
 Read [this article][instanceof-vs-array-is-array] for more information.
 
+## Array Methods
+
+Some of the [methods][array_methods] that are available on `Array.prototype` can be used to add or remove from the array. Here are a few of them:
+
+### push
+
+> The `push()` method adds one or more elements to the end of an array and returns the new length of the array.<sup>1</sup>
+
+```javascript
+const names = ['Jack', 'Laura', 'Paul', 'Megan'];
+names.push('Jill'); // => 5
+names;
+// => ['Jack', 'Laura', 'Paul', 'Megan', 'Jill']
+```
+
+### pop
+
+> The `pop()` method removes the last element from an array and returns that element. This method changes the length of the array.<sup>2</sup>
+
+```javascript
+const names = ['Jack', 'Laura', 'Paul', 'Megan'];
+names.pop(); // => 'Megan'
+names;
+// => ['Jack', 'Laura', 'Paul']
+```
+
+### shift
+
+> The `shift()` method removes the first element from an array and returns that removed element. This method changes the length of the array.<sup>3</sup>
+
+```javascript
+const names = ['Jack', 'Laura', 'Paul', 'Megan'];
+names.shift(); // => 'Jack'
+names;
+// => ['two', 3, 'four']
+```
+
+### unshift
+
+> The unshift() method adds one or more elements to the beginning of an array and returns the new length of the array.<sup>4</sup>
+
+```javascript
+const names = ['Jack', 'Laura', 'Paul', 'Megan'];
+names.unshift('Jill'); // => 5
+names;
+// => ['Jill', 'Jack', 'Laura', 'Paul', 'Megan']
+```
+
+### splice
+
+> The splice() method changes the contents of an array by removing or replacing existing elements and/or adding new elements in place.<sup>5</sup>
+
+```javascript
+const names = ['Jack', 'Laura', 'Paul', 'Megan'];
+names.splice(2, 1, 'Jill');
+names;
+// => ['Jack', 'Laura', 'Jill', 'Megan']
+```
+
+[1] push, MDN. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push (referenced September 29, 2021)
+
```

Comment:
```suggestion
```

Reply:
I don't know if it works without empty lines, but if it does, can we do it without so it looks like the other references?

Reply:
If you we're doing footnote, shouldn't it have been `[^1]` instead of `<sup>1</sup>`?